### PR TITLE
Implement "Move Users" dialog

### LINKED ISSUE #1833: Add Move users dialog to Qt client
### Description

Currently, the only way to move users in Qt client is to select users on channel tree than move it to channel we wish.
In classic client, a dialog was present to allow moving users quickly. A similar dialog should be added to Qt client with a list of checkboxes to choose users to move, and a combobox to choose channel.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

### PR DESCRIPTION
**Pull request by CoBC**
Fixes #1833
@bear101 Maybe you prefer to have an ui file. If yes, can you create it please? :)

### CODE REVIEW

File: Client/qtTeamTalk/moveusersdlg.cpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+void MoveUsersDlg::setupUi()
+{
+    setWindowTitle(tr("Move Users"));
+    setWindowIcon(QIcon(APPICON));
+
+    QVBoxLayout* mainLayout = new QVBoxLayout(this);
+
+    QGroupBox* usersGroup = new QGroupBox(tr("Users"), this);
+    QVBoxLayout* usersLayout = new QVBoxLayout(usersGroup);
```
Comment (bear101) [inline Client/qtTeamTalk/moveusersdlg.cpp:40]: Clang-Tidy says "usersLayout" may memory leak. I just can't figure out why. Can you? It appears to be the case if userIds.size() is 0. However, if I put a check for userIds.size()>0 then it still complains :(

**Review by bear101 — COMMENTED** [review]
Btw, I've added one commit

File: Client/qtTeamTalk/moveusersdlg.cpp (hunk 1 above)
Comment (CoBC) [reply Client/qtTeamTalk/moveusersdlg.cpp:40]: Maybe just add a QLabel if userids is empty to indicates there is no user available. This QLabel will never be visible but maybe clang-tidy will be happy. Not sure :)

Comment (bear101) [reply Client/qtTeamTalk/moveusersdlg.cpp:40]: Hm, that also didn't work (with the QLabel). I'll merge. It looks like a false-positive.

**Comment by bear101** [comment]
btw, .ui files are not good for dynamic content.

**Pull request merged**